src/Symfony/Component/BrowserKit/Tests/AbstractBrowserTest.php

```diff
@@ -294,12 +295,8 @@ public function testClickLinkNotFound()
         $client->setNextResponse(new Response('<html><a href="/foo">foobar</a></html>'));
         $client->request('GET', 'http://www.example.com/foo/foobar');
 
-        try {
-            $client->clickLink('foo');
-            $this->fail('->clickLink() throws a \InvalidArgumentException if the link could not be found');
-        } catch (\Exception $e) {
-            $this->assertInstanceOf(\InvalidArgumentException::class, $e, '->clickLink() throws a \InvalidArgumentException if the link could not be found');
-        }
+        $this->expectException(\InvalidArgumentException::class);
```

Comment:
```suggestion
        $this->expectException(InvalidArgumentException::class);
```
?

Reply:
No, this exception comes from the client, not browserkit.

Reply:
to be exact, it comes from the crawler, not from the client.